exporter/mezmoexporter/README.md

```diff
@@ -0,0 +1,30 @@
+# Mezmo Exporter
+
+This exporter supports sending OpenTelemetry log data to [LogDNA (Mezmo)](https://logdna.com).
+
+# Configuration options:
+
+- `ingest_url` (required): Use the default URL of `https://logs.logdna.com/log/ingest` unless a custom URL has been provided for your account.
+- `ingest_key` (required): Ingestion key used to send log data to LogDNA.  See [Ingestion Keys](https://docs.logdna.com/docs/ingestion-key) for more details.
+
+# Example:
+## Simple Log Data
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: ":4317"
+
+exporters:
+  mezmo:
+    ingest_url = "https://logs.logdna.com/log/ingest"
```

Comment:
These values should be assigned with `:` instead of `=`

Reply:
Fixed.